dask_kubernetes/kubernetes.yaml

```diff
@@ -7,7 +7,6 @@ kubernetes:
   host: "0.0.0.0"
   port: 0
   env: {}
-  scheduler-timeout: "5 minutes" # Length of inactivity to wait before closing the cluster
```

Comment:
I'm unsure about grabbing the config option from the distributed settings. It may cause confusion not having it in the kubernetes config.

Reply:
sounds good. I hadn't realized that the `dask_kubernetes.Scheduler` class bypasses `distributed.Scheduler` completely (I think?), so agree this makes sense.